### DISCUSSION TITLE
psrp prefix before the domain, Sender address rejected: Domain not found

### DISCUSSION THREAD

**Discussion by ofekifergan** [postalserver/postal#3244] [Help with using Postal]
Hi,

I am trying to send some emails from `test@oss.com` (fake name)

The domain is authenticated in postal but whenever I try o send mails I get Soft fail (and later Hard fail):
```
452 4.1.8 <test@psrp.oss.com>: Sender address rejected: Domain not found
```

I told my client to exclude the email origin IP address and he still gets this error.
Is this because we told him that the mail will come from `test@oss.com` but eventually it comes from `test@psrp.oss.com`?

Can you explain the importance of the psrp prefix coming before the domain?
Does our clients needs to exlude something in order to make this work?

**Comment by willpower232** [comment]
psrp is the return path of the domain, you should see it being checked when adding the domain to Postal.

It is basically used to deliver slower bounces and other rejections which don't happen immediately when the email is given to the receiving server.

It is automatically added to the email as it gets sent by Postal.

Have you definitely set it up? I presume you're seeing that error message in the Postal UI when you look at the message?

**Reply by ofekifergan** [reply]
Yes, I can see it in Postal ui.

My question is if I am sending a mail from `test@oss.com` doesn't `test@psrp.oss.com` different from what I inteded?
This can be a problem at stricted recpients mail servers (if mail server managers want to exlude domains for example)?
I say that I want to send an email from `test@oss.com`, but actually I send from `test@psrp.oss.com`.

**Reply by willpower232** [reply]
I would presume that whatever is receiving the email is either trying to deliver a bounce message or verifying that the return path is valid before allowing the email address to continue to its recipient.

Whoever receives the message will see it as you expect but the server needs to know how to appropriately return any problems.

**Reply by ofekifergan** [reply]
What can I do to help the server validate?
I need to tell the customers to exclude `psrp.oss.com` so they can check the return path CNAME?

**Reply by willpower232** [reply]
If the DNS record is set up for it, as per the domain page in the Postal UI, then do you have port 25 open to the internet?

**Reply by Rez0k** [reply]
I have port 587 open,
Can I delete the psrp in the sender domain?
How can I achieve that?
